<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="8" hostname="john-PC" name="nov13.Orange" tests="14" failures="0" timestamp="14 Nov 2019 03:36:05 GMT" time="16.977" errors="2">
  <testcase name="verifylogin" time="2.944" classname="nov13.Orange"/>
  <testcase name="verifylogin" time="7.254" classname="nov13.Orange"/>
  <testcase name="verifylogin" time="0.000" classname="nov13.Orange">
    <skipped/>
  </testcase> <!-- verifylogin -->
  <testcase name="verifylogin" time="0.000" classname="nov13.Orange">
    <skipped/>
  </testcase> <!-- verifylogin -->
  <testcase name="verifylogin" time="0.000" classname="nov13.Orange">
    <skipped/>
  </testcase> <!-- verifylogin -->
  <testcase name="setup" time="0.000" classname="nov13.Orange">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at nov13.Orange.setup(Orange.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setup -->
  <testcase name="verifylogin" time="0.000" classname="nov13.Orange">
    <skipped/>
  </testcase> <!-- verifylogin -->
  <testcase name="verifylogin" time="0.000" classname="nov13.Orange">
    <skipped/>
  </testcase> <!-- verifylogin -->
  <testcase name="verifylogin" time="2.993" classname="nov13.Orange"/>
  <testcase name="verifylogin" time="0.000" classname="nov13.Orange">
    <skipped/>
  </testcase> <!-- verifylogin -->
  <testcase name="setup" time="0.237" classname="nov13.Orange">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at nov13.Orange.setup(Orange.java:45)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setup -->
  <testcase name="verifylogin" time="0.000" classname="nov13.Orange">
    <skipped/>
  </testcase> <!-- verifylogin -->
  <testcase name="verifylogin" time="3.549" classname="nov13.Orange"/>
  <testcase name="verifylogin" time="0.000" classname="nov13.Orange">
    <skipped/>
  </testcase> <!-- verifylogin -->
</testsuite> <!-- nov13.Orange -->
